requirements.txt

```diff
@@ -0,0 +1,6 @@
+ray[rllib]==2.4.0
+ray[tune]
+matplotlib
+qutip
+torch
+tensorboard
```

Comment:
Are these required for the tests? Even `tensorboard`? 

Probably fine, but I've been burned in the past trying to install tensorflow/tensorboard stuff where it ends up changing versions of other packages and breaking everything

Reply:
tensorboard is not required and neither is matplotlib. The others are needed or will be needed.

Reply:
deleted